src/ctelnet.cpp

```diff
@@ -1550,8 +1560,14 @@ void cTelnet::loadReplay(QString& name)
         postMessage(msg);
         replayStream.setDevice(&replayFile);
         loadingReplay = true;
-        mudlet::self()->replayStart();
-        _loadReplay();
+        if (mudlet::self()->replayStart()) {
+            _loadReplay();
+        } else {
+            loadingReplay = false;
```

Comment:
This breaks something - if you have a replay running and you use `loadRawFile()`, the replay pauses and the buttons get stuck.

Reply:
OH BU****! I had forgotten that it was possible to start a __replay__ from a lua script... 🤦‍♂️ I'll have to have a think about that (how did it start out being called `loadRawFile` rather than `loadReplay`?)